docs/quickstarts/outlook-quickstart.md

```diff
@@ -109,6 +105,10 @@ The add-in project that you've created with the Yeoman generator contains sample
     > [!NOTE]
     > If you receive the error "We can't open this add-in from localhost" in the task pane, follow the steps outlined in the [troubleshooting article](/office/troubleshoot/office-suite-issues/cannot-open-add-in-from-localhost).
```

Comment:
yo office now has a prompt during setup that mentions setting the loopback exemption. maybe we can tell users to say yes to that?

Reply:
I've made a note to address this in another PR and added it to our sync agenda to discuss.